### PR TITLE
tika: add `enableOcr` option

### PR DESCRIPTION
**Pull request by drupol**
Following `nixpkgs` PR @ https://github.com/NixOS/nixpkgs/pull/327886, this PR add the `enableOcr` option.

### CODE REVIEW

**Comment by shivaraj-bh** [comment]
As https://github.com/NixOS/nixpkgs/pull/327886 will take a few days before it reaches `nixpkgs-unstable`, let’s wait for it (`tika` tests will fail otherwise)

**Comment by drupol** [comment]
`nixpkgs-unstable` has just moved on, this should be good now.

File: nix/services/tika.nix [hunk 1]
```diff
@@ -33,21 +41,27 @@ in
...
-            command = "${lib.getExe config.package} --host ${config.host} --port ${toString config.port} ${lib.optionalString (config.configFile != null) "--config ${config.configFile}"}";
-            availability.restart = "on_failure";
-            readiness_probe = {
-              http_get = {
-                host = config.host;
-                port = config.port;
+          "${name}" =
+            let
+              tikaPackage = config.package.override {
+                enableOcr = config.enableOcr;
```
Comment (drupol) [inline nix/services/tika.nix:47]:
```suggestion
                inherit (config) enableOcr;
```

**Comment by shivaraj-bh** [comment]
Thanks!

**Pull request merged**